profile.html

```diff
@@ -26,55 +32,74 @@
                 </div>
 
                 <div class="menuElements">
+                  <ul>
                     <li class="communityPage_title">Community</li>
-                    <li><a id="login">Logout</a></li>
-                    <li><a id="signUp">Profile</a></li>
-                    <li><a href="contact.asp">Settings</a></li>
-                    <li><a id="aboutPage">About</a></li>
+                    <li><a id="login" href="#">Logout</a></li>
+                    <li><a id="signUp" href="#" >Profile</a></li>
+                    <li><a href="settings.html" >Settings</a></li>
+                    <li><a id="aboutPage" href="#" >About</a></li>
                     <!-- have a search bar here  -->
+                  </ul>
                 </div>
             </ul>
-
-        </div>
-
-        <div class="profilePage_userDetails">
-            <h2>Mercy Nekesa</h2>
-            <p id="cityCountry">kampala,Uganda</p>
-        </div>
-        <div class="profilePage_title">
-            <h1>story</h1>
-            <p>Time</p>
-        </div>
-        <div class="profilePage_archivedPosts">
-            <ul>
-                <li> <a href=""> Past Posts</a></li>
-                <li> <a href=""> Saved Posts</a> </li>
-            </ul>
         </div>
 
-        <img class="profilePage_clock" src="images\profilecassette.png" alt="clock">
-
-
-
-        <div class="profilePage_pastContent">
+            <div class="profilePage_title">
+                <h1>Profile</h1>
+            </div>
+      </header>
 
-            <p>home for past and saved content</p>
+        <!--<img class="profilePage_clock" src="images\profilecassette.png" alt="clock">-->
```

Comment:
delete this since it's code that is not needed anymore

Reply:
done